### DISCUSSION TITLE
可以在sftp中选择复制文件路径吗？

### DISCUSSION THREAD

**Discussion by fly9593** [Eugeny/tabby#6780] [Q&A]
![图片](https://user-images.githubusercontent.com/59961051/179141956-3bb71bc2-f5ee-485a-9f63-a01a5260ad13.png)
如图，在sftp中点击右键只有新建文件夹和删除两个选项，有没有像是复制，粘贴，复制文件路径的功能呢？

**Comment by MartinLau7** [comment]
我建议你用英文提问，你不可能指望每个人都会中文 [#5532](https://github.com/Eugeny/tabby/issues/5532)
